### PR TITLE
chore: add REUSE tool

### LINKED ISSUE #6: [rl-reuse_tool-2] Violation against OSS Rules of Play
A violation against the OSS Rules of Play has been detected.

Rule ID: rl-reuse_tool-2
Explanation: Does it have LICENSES directory with licenses? **No**

Find more information at: https://sap.github.io/fosstars-rating-core/oss_rules_of_play_rating.html

### LINKED ISSUE #8: [rl-reuse_tool-1] Violation against OSS Rules of Play
A violation against the OSS Rules of Play has been detected.

Rule ID: rl-reuse_tool-1
Explanation: Does README mention REUSE? **No**

Find more information at: https://sap.github.io/fosstars-rating-core/oss_rules_of_play_rating.html

### PR DESCRIPTION
**Pull request by SebastianWolf-SAP**
In line with SAP OSPO requirements, this PR integrates the REUSE tool

**Note:** Please register the repository with the REUSE Tool to complete the integration (see #9 and #10)

Fixes #6
Fixes #8

### CODE REVIEW

File: LICENSES/Apache-2.0.txt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+END OF TERMS AND CONDITIONS
+
+APPENDIX: How to apply the Apache License to your work.
+
+To apply the Apache License to your work, attach the following boilerplate notice, with the fields enclosed by brackets "[]" replaced with your own identifying information. (Don't include the brackets!)  The text should be enclosed in the appropriate comment syntax for the file format. We also recommend that a file or class name and description of purpose be included on the same "printed page" as the copyright notice for easier identification within third-party archives.
+
+Copyright [yyyy] [name of copyright owner]
```
Comment (yamladenov) [inline LICENSES/Apache-2.0.txt:61]: Placeholders should be replaced

**Review by yamladenov — CHANGES_REQUESTED** [review]

File: LICENSES/Apache-2.0.txt (hunk 1 above)
Comment (SebastianWolf-SAP) [reply LICENSES/Apache-2.0.txt:61]: Done

File: LICENSES/Apache-2.0.txt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+END OF TERMS AND CONDITIONS
+
+APPENDIX: How to apply the Apache License to your work.
+
+To apply the Apache License to your work, attach the following boilerplate notice, with the fields enclosed by brackets "[]" replaced with your own identifying information. (Don't include the brackets!)  The text should be enclosed in the appropriate comment syntax for the file format. We also recommend that a file or class name and description of purpose be included on the same "printed page" as the copyright notice for easier identification within third-party archives.
+
+Copyright 2016-2021 SAP SE or an SAP affiliate company and cloud-metrics-dashboard contributors
```
Comment (yamladenov) [inline LICENSES/Apache-2.0.txt]: Maybe this should be also added in the LICENSE file

**Review by yamladenov — CHANGES_REQUESTED** [review]

File: LICENSES/Apache-2.0.txt (hunk 2 above)
Comment (SebastianWolf-SAP) [reply LICENSES/Apache-2.0.txt]: Actually, it should and this change should be reverted as it's not needed here. See the discussion in https://github.com/SAP/cloud-metricswatcher/pull/12#pullrequestreview-730451929 I'll do that and send it back to review afterwards

Comment (yamladenov) [reply LICENSES/Apache-2.0.txt]: That's great thanks

**Pull request merged**